### PR TITLE
Issue #267 - Displaying decimals on Watch for mmol

### PR DESCRIPTION
**Pull request by alternateal**
Updated code to use significant digits to enable decimals to be shown on both watch app and complication.

### CODE REVIEW

File: WatchApp Extension/ComplicationController.swift [hunk 1]
```diff
@@ -40,10 +40,13 @@ final class ComplicationController: NSObject, CLKComplicationDataSource {
     
     // MARK: - Timeline Population
 
-    private lazy var formatter = NumberFormatter()
+ //   private lazy var formatter = NumberFormatter()
 
     func getCurrentTimelineEntry(for complication: CLKComplication, withHandler handler: (@escaping (CLKComplicationTimelineEntry?) -> Void)) {
 
+        let formatter = NumberFormatter(); formatter.usesSignificantDigits = true; formatter.minimumSignificantDigits = 2; formatter.maximumSignificantDigits = 3
```
Comment (ps2) [inline WatchApp Extension/ComplicationController.swift]: What does this display with mg/dl?

Need to avoid repeating this code three times.

**Comment by ps2** [comment]
Actually, I think we will be able to use @bharat 's extension to do this: https://github.com/LoopKit/Loop/pull/281/commits/b41ceda7b6f7a3c90261e62a854f273297da1419

**Comment by alternateal** [comment]
Happy to use @bharat's extension (provided it works on watch also) - I knew it was quick and dirty solution - just needed to make something workable for users of mmol/L. Happy to close the thread if you want?

**Comment by ps2** [comment]
I would rather you keep it open and then update it to use the HKUnit unit extension when it's available!

**Comment by loudnate** [comment]
Once the HKUnit extension is available, include it in the watch target along with `NSNumberFormatter.swift`, so you can use `NumberFormatter.glucoseNumberFormatter(for: HKUnit)`

**Comment by ps2** [comment]
The HKUnit extension has been merged into dev, if you want to rebase and update this PR to use it.

**Comment by ps2** [comment]
I believe this is resolved properly with #379

**Pull request closed without merging**